Node.js Code Samples - PostgreSQL - Remove Node Install

burkeholland opened the pull request:
I think that we can assume that if people are looking at a Node guide, they already have Node installed. It seems OK to me to just have it listed as a prerequisite.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

burkeholland committed:
Node.js Code Samples - PostgreSQL - Remove Node Install

I think that we can assume that if people are looking at a Node guide, they already have Node installed. It seems OK to me to just have it listed as a prerequisite.
